android/src/main/res/values/breakdown2016_styles.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="breakdown_base">
```

Comment:
This could include `layout_columnSpan`, `layout_columnWeight`, `layout_rowSpan`, `layout_gravity`; it looks like those are consistent across all the `GridView`'s children. You could shave a few hundred lines off the layout file this way.